### ISSUE TITLE
Override the default StatusBar color.

### ISSUE THREAD

**Issue by kayode0x** [kanelloc/react-native-animated-header-scroll-view#18]
I was trying to set a custom StatusBar color because my app has a theme, but it refused to update. I thought I was doing something wrong, but from debugging, I saw that you have a fixed color for the StatusBar [on this line](https://github.com/kanelloc/react-native-animated-header-scroll-view/blob/main/src/components/AnimatedHeader.tsx#L22).

TLDR: Are there any intentions to make it extensible? like p ass in light | dark as StatusBar props.

I also noticed that the lib is light themed by default; it would be nice if we could pass in colors as parameters, with fallbacks.

Right now, I want to patch the package to fit my needs; please let me know if you intend to integrate this, so that I may fork and create a PR.

Thank you for the wonderful package!

**Comment by kanelloc** [comment]
Hey @kayode0x. I created the above PR to reflect this need.

> TLDR: Are there any intentions to make it extensible? like p ass in light | dark as StatusBar props.

There is no need to have a `<StatusBar/>` component in `HeaderNavBar.tsx` so I just removed it. 😆 

You can use a custom `StatusBar` component as shown here: https://github.com/kanelloc/react-native-animated-header-scroll-view/pull/19/files#diff-be242f52a0eafa418ba71d98e623bf3fc97359c9179b47cfa791f1453293434eR14

> I also noticed that the lib is light-themed by default; it would be nice if we could pass in colors as parameters, with fallbacks.
What do you mean by `light themed`?
